docker-compose/docker-compose-certify/config/esignet-default.properties

```diff
@@ -491,12 +491,12 @@ mosip.esignet.vci.key-values={\
               'credential_issuer': '${mosip.esignet.vci.identifier}',   \
               'credential_endpoint': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', \
               'display': {{'name': 'Insurance', 'locale': 'en'}},\
-              'credentials_supported' : { \
```

Comment:
authorization_servers availability is missing, please check this also in all the property files

Reply:
I've updated all the well-known configs in test & esignet-default.properties file as suggested.